disablestrongnamevalidation.ps1

```diff
@@ -0,0 +1,2 @@
+& "C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.6.2 Tools\x64\sn.exe" -Vr *,31bf3856ad364e35
+& "C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.6.2 Tools\sn.exe" -Vr *,31bf3856ad364e35
```

Comment:
@pharring Thank-you very much for the explanation, that explains the behavior we were seeing.